### DISCUSSION TITLE
Not Sync with Beat at Input Position

### DISCUSSION THREAD

**Discussion by jefflim69** [melanchall/drywetmidi#234] [Q&A]
![Beat_Sync](https://user-images.githubusercontent.com/12184282/207759751-28a5119e-6c6d-4bad-95a4-0128476a1feb.png)
[](url)

<OS> PC , Windows 10
<Tool> Unity 3d , 2020.2.7f1
I'm using DryWetMIDI **Unity Sample** from here .(Not know well DryWetMIDI version)
Here I include the Unity project . There my Midi Files and Audio included in WorkScene .

https://drive.google.com/file/d/1YCtrpQiFRfJdbKv1kpSMq_uhf3qytbxB/view?usp=share_link
My Email : jefflim69@hanmail.net   or jefflim0326@gmail.com

I Tested other Songs, But the End of Bottom Position is Perfect Position of Input .
How Can I change the Input Position , and Sync with beat .
in Short word I want Change the Check Line to more move Up and Input beat to Sync Audio Rhythm.
Now, is little fast than Audio beat .

Thanks a lot about your work .

**Comment by melanchall** [comment]
Please read this first: https://melanchall.github.io/drywetmidi/articles/dev/Support.html

**Reply by jefflim69** [reply]
I read that , and edit the text , would you answer me about my question ?

**Reply by melanchall** [reply]
Unfortunately it's pretty much amount of work to run your project and to understand what's going on. I can handle it in next year only (at the start of January). Is it OK?

**Reply by jefflim69** [reply]
I Solved this with DAW(Cakewalk).
I Edit the notes move back to when I wanted timing .
Thank you .
